igvm/hypervisor_preferences.py

```diff
@@ -179,25 +178,23 @@ class to let them be prepared lazily.  When Python needs to compare
     executed for that hypervisor by LazyCompare.
 
     The LazyCompare optimization has one other benefit of bring more
-    visibility about selection.  After the sorting is done, we can check
+    visibility about the selection.  After the sorting is done, we can check
     which preferences are actually executed for the selected hypervisor,
-    and log which preference caused this hypervisor to be sorted before
-    the next one.
+    and log which preference caused this hypervisor to be sorted after
+    the previous or before the next one.
     """
-    log.debug('Sorting hypervisors by preference...')
+    plen = len(preferences)
```

Comment:
len() on lists is O(1). Let's just re-query it instead of having this obscurely named variable.